tests/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+#
+# Development configuration of dependent services for Docker Compose.
+#
+postgresql:
+    image: postgres:9.4
+    environment:
+        POSTGRES_USER: resolwe
+        POSTGRES_DB: resolwe
+    ports:
+        - "55432:5432"
```

Comment:
I don't know. My idea was that I can only run docker-compose in genesis-base and run tests of resolwe and resolwe-bio in their corresponding test projects directly in the same docker container. So that I only run one DB, but can work with different repos separately.